product_docs/docs/epas/14/epas_inst_linux/installing_epas_using_edb_repository/index.mdx

```diff
@@ -16,7 +16,6 @@ For platform-specific install instructions, including accessing the repo, see:
 
 - Linux x86-64 (amd64):        
     - [RHEL 8/OL 8](x86_amd64/epas_rhel8_x86)
-    
```

Comment:
I added these blank lines on purpose to add spacing between the list items in the output